public/data/ZainMhrez.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Zain Mhrez",
+  "type": "personal",
+  "bio": "Jr Frontend developer | Reactjs, learning typescript and web3",
+  "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ZainMhrez",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/zainmhrez1",
+      "icon": "twitter"
+    },
+	{
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/zain-mhrez/",
+      "icon": "linkedin"
+    }
+  ],
```

Comment:
There's an extra comma in here.
```suggestion
  ]
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable